src/templates/failed.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{% block content %}
+<div class="container">
+    <div class="row">
+        <div class="col-5">
+            <h1>Failed!</h1>
+            <p>Something went wrong when signing you up!</p>
+            <a href={{ url_for('subscribe') }}><button type="button"  id='tryagain' class="btn btn-lg btn-primary btn-dark">Try again -></button></a>
```

Comment:
The url should always be between quotes (it still works but it can give errors)
So `<a href="{{ url_for('subscribe') }}">`

Same for all other usages